.github/workflows/publish.yaml

```diff
@@ -0,0 +1,51 @@
+jobs:
+  release_pypi:
+      name: Build and publish Python distributions 📦 to PyPI and TestPyPI
+      runs-on: ubuntu-latest
+      if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
+      steps:
+        - uses: actions/checkout@v3
+        - name: Set up Python 3.10
+          uses: actions/setup-python@v4
+          with:
+            python-version: "3.10"
+            cache: 'pip'
+        - name: Install pypa/build
+          run: |
+            pip install --upgrade build
+        - name: Build a binary wheel and a source tarball
+          run: |
+            python -m build --sdist --wheel --outdir dist/
+        - name: Publish distribution 📦 to Test PyPI
+          uses: pypa/gh-action-pypi-publish@f8c70e705ffc13c3b4d1221169b84f12a75d6ca8
+          with:
+            password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+            repository_url: https://test.pypi.org/legacy/
+        - name: Publish distribution 📦 to PyPI
+          uses: pypa/gh-action-pypi-publish@f8c70e705ffc13c3b4d1221169b84f12a75d6ca8
+          with:
+            password: ${{ secrets.PYPI_API_TOKEN }}
+  release_conda:
+    name: Build and publish conda package
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
+    steps:
+    - uses: actions/checkout@v3
+    - uses: mamba-org/setup-micromamba@875557da4ee020f18df03b8910a42203fbf02da1
+      with:
+        init-shell: bash
+        cache-environment: true
+      - name: Install build tools
+        run: |
+          micromamba install -y anaconda-client boa -c conda-forge
+      - name: Make build folder and build package
+        run: |
+          mkdir ${{ github.workspace }}/build
+          conda mambabuild --croot ${{ github.workspace }}/build recipes -c conda-forge
+      - name: Upload to conda
+        run: |
+          BUILD_DIR=${GITHUB_WORKSPACE}/build
+          BUILD=$(find "$BUILD_DIR" -name '*.tar.bz2')
+          anaconda --token "$ANACONDA_TOKEN" upload --user openghg --label main "$BUILD"
+        env:
+          ACCESS_TOKEN: ${{ secrets.ANACONDA_TOKEN }}
```

Comment:
Indicating this needs a new line at the end of the file.

Reply:
OK fixed.